addons/html_builder/static/src/core/grid_layout/grid_layout_plugin.js

```diff
@@ -100,12 +100,12 @@ export class GridLayoutPlugin extends Plugin {
         if (!this.config.isMobileView(this.overlayTarget)) {
             buttons.push(
                 {
-                    class: "o_send_back oi",
+                    class: "oi-move-down oi",
                     title: _t("Send to back"),
                     handler: this.sendGridItemToBack.bind(this),
                 },
                 {
-                    class: "o_bring_front oi",
```

Comment:
Instead of removing the old classes, I would have only removed the CSS associated to them and added your new classes, as they were used to target these specific grid buttons easily in JS too (and they matched the button handlers) :thinking: (its like the clone button: it uses `fa fa-clone` but still has `oe_snippet_clone` to identify it :+1: )

If you rename them as asked in my other comment, I guess it could be fine, but I still think they should be easily "indentifiable" so let's keep the old classes :slightly_smiling_face: